### ISSUE TITLE
Lighthouse Accessibility

### ISSUE THREAD

**Issue by MJOW1999** [fac22/week3-BDJS#46]
You have a really good lighthouse accessibility score of 93 💡

The only minor issue  was that: 'List items (`<li>`) are not contained within `<ul>` or `<ol>` parent elements.'

This means that screen readers would not announce them properly
